packages/@uppy/dashboard/src/components/FileItem/FileProgress/index.scss

```diff
@@ -68,7 +68,7 @@ svg.UppyIcon.retry {
 }
 
 
-// Svg styles that depend upon the state of the file.
```

Comment:
Seems grammatically there’s no difference? https://english.stackexchange.com/questions/34291/depend-upon-or-depend-on ;)

Reply:
Thank you, I'd never google!